bbc/simorgh
# WSTEAM1-370 - Media Article Page dark UI (test only)

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-370 

Overall changes
======
- Applies dark UI to the Media Article Page page type
- This change will be restricted to the Test environment until a11y and UX checks have been completed and signed off

Code changes
======
- Adds checks for `isDarkUi` to various components used in the Media Article Page page type
- Updates Storybook for MediaArticlePage to set the correct `pageType` and move it into its own section

Testing
======
1. Navigate to https://darkui-with-bool-flag--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-media-article-page--media-article-page
2. Confirm that the UI is now using the dark colour palette
3. _Note_ some UI elements may not have the correct colouring, e.g. the right hand column. This is deliberate, as these items will be removed in favour of the new 'Latest media' promo items.

a11y swarm
======
This will require a dev a11y swarm. Colours have been signed off, but implementation could do with a check to ensure colour contrast is correct

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
